page_objects/Base_Page.py

```diff
@@ -124,6 +122,7 @@ def get_current_driver(self):
     def register_testrail(self):
         "Register TestRail with Page"
         self.testrail_flag = True
+        from utils.Test_Rail import Test_Rail  # pylint: disable=import-error,import-outside-toplevel
         self.tr_obj = Test_Rail()
```

Comment:
Can you rename this variable to `self.testrail_object`?

Reply:
self.tr_obj = Test_Rail()
 
This was existing before and not a new change .  I think it follows as TestRail as tr . If we proceed with this update, we might need to rename numerous locations to preserve consistency throughout the framework Basepage, Mobile Base Page, and setup_testrail.

Reply:
Yeah Drishya, we seem to be persisting with the bad variable name.  Let's change it as part of this PR.

Reply:
I ran a `grep` to find out where we have used `self.tr_obj`, we seem to be using it in:
- `./page_objects/Base_Page.py:        self.tr_obj = Test_Rail()`
- `./page_objects/Base_Page.py:            self.tr_obj.update_testrail(case_id,test_run_id,result_flag,msg=msg)`
- `./page_objects/Mobile_Base_Page.py:        self.tr_obj = Test_Rail()`
- `./page_objects/Mobile_Base_Page.py:            self.tr_obj.update_testrail(case_id,test_run_id,result_flag,msg=msg)`

Reply:
Done!